runtime/src/main/java/com/tietoevry/quarkus/resteasy/problem/postprocessing/ProblemRecorder.java

```diff
@@ -24,4 +25,9 @@ public void enableMetrics() {
         ExceptionMapperBase.postProcessorsRegistry.register(new MicroprofileMetricsCollector());
```

Comment:
Since a user should not call `ExceptionMapperBase.postProcessorsRegistry.register` directly, maybe it makes sense reducing the visibility of that particular method.

Reply:
It's not that you should not call it - it's not forbidden, but you're right - it could have been package private. As such change is breaking, we can do it in 4.0 earliest though.

Reply:
Ah ok, I thought your intention was not to offer this API publicly. But yeah, fine to me.

Reply:
It doesn't really matter now what my intention was, it would be api breaking change, and - if we want to be predictable and compliant with semver - we can't change it `just like that`